e2e/loadbalancer_test.go

```diff
@@ -79,6 +91,7 @@ func TestLoadbalancerProxy(t *testing.T) {
 			Namespace: "default",
 			Annotations: map[string]string{
 				"kubernetes.civo.com/loadbalancer-enable-proxy-protocol": "send-proxy",
+				"kubernetes.civo.com/firewall-id":                        e2eTest.cluster.FirewallID,
```

Comment:
>[!NOTE]
Same as the below comment.
https://github.com/civo/civo-cloud-controller-manager/pull/37/files#r2031178002